src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -829,6 +833,15 @@ private static final String getIncrementalSearchMessage()
       return sIncrementalSearchMessage_;
    }
    
+   private void withCurrentlyFocusedEditor(CommandWithArg<DocDisplay> command)
+   {
+      InputEditorDisplay consoleEditor = consoleEditorProvider_.getConsoleEditor();
+      if (consoleEditor != null && consoleEditor.isFocused() && consoleEditor instanceof DocDisplay)
```

Comment:
Note that the null check here is used as e.g. Satellite windows won't necessarily have access to the console editor.